### DISCUSSION TITLE
<title>Why I'm getting 404 error

### DISCUSSION THREAD

**Discussion by Vamsi917** [community/community#68443] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

i have mulltiple css,html files uploades as single folder bootsrap,
I created index.html in which I gave as <ahref="bs1.html">task1>/a>, <ahref="bs2.html">task2>/a> and so on but when I open this link its not 404 not found why?
repository link:https://github.com/Vamsi917/Bootsrap.git
profile:Vamsi917

**Comment by yashSingh97** [comment]
```
<a href="./bootsratp/bs1.html">task1</a>
<a href="../bootsratp/bs2.html">task2</a>
<a href="../bootsratp/bs3.html">task3</a>
```
try this

**Comment by yashSingh97** [comment]
can you see if this answers ur question?

**Reply by Vamsi917** [reply]
didn't worked broo

**Reply by yashSingh97** [reply]
worked on my end, dunno what you are doing wrong
edit: it seems u had updated ur repo yesterday, i just git clone ur repo and ran it on my end without any modifications and it runs fine lol, redirects to task1-3 without any errors
edit2: also i just noticed i didn't copied the right directories (. ..), so anyways the first bs1 should have worked
